configs/go.json

```diff
@@ -122,6 +122,9 @@
       ],
       "deploy_versioning": false
     },
+    {
+      "target": "data-preservation-programs/singularity"
```

Comment:
@xinaxu  for this to function @web3-bot needs to be given write permissions to the repo.

Reply:
Invite sent to @web3-bot

Reply:
Thank you, and sorry for the delay. Here's the uCI PR - https://github.com/data-preservation-programs/singularity/pull/78